pkgs/misc/themes/gtk3/paper-gtk-theme/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, gtk }:
+
+stdenv.mkDerivation rec {
+  version = "6a5f14cfe697b0a829456a1fd560acdcddc6043f";
+  name = "paper-gtk-theme-${version}";
+
+  src = fetchFromGitHub {
+    owner = "snwh";
+    repo = "paper-gtk-theme";
+    sha256 = "0kyn3a6sq0z22vds6szl630jv20pjskjbdg0wc8abxzwg0vwxc5m";
+    rev = version;
+  };
+
+  dontBuild = true;
+  installPhase = ''
+    mkdir -p $out/share/themes/Paper/
+    cp -r ./Paper/ $out/share/themes/
+  '';
+
+  preferLocalBuild = true;
+
+  meta = with stdenv.lib; {
+    description = "Paper is a modern desktop theme suite. Its design is mostly flat with a minimal use of shadows for depth.";
```

Comment:
It's better not to mention the package name in the description (see http://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes), also please shorten it to a single sentence and drop the full stop at the end. Thanks!